doc/changes/names.inc

```diff
@@ -575,3 +575,5 @@
 .. _Zhi Zhang: https://github.com/tczhangzhi/
 
 .. _Zvi Baratz: https://github.com/ZviBaratz
+
+.. _Gonzalo Reina: https://greina.me/
```

Comment:
Thanks for adding yourself here!  We try to keep this file alphabetized, can you move your name up to the right place?

Also: need a changelog entry in `doc/changes/devel.rst`